pkg/kudoctl/cmd/init.go

```diff
@@ -118,13 +120,16 @@ func (initCmd *initCmd) validate(flags *flag.FlagSet) error {
 	if flags.Changed("wait-timeout") && !initCmd.wait {
 		return errors.New("wait-timeout is only useful when using the flag '--wait'")
 	}
+	if initCmd.webhooks != "" && initCmd.webhooks != "InstanceValidation" {
+		return errors.New("webhooks can be only empty or contain a single string 'InstanceValidation'. No other webhooks supported right now")
```

Comment:
again... I know we are looking to land this and it is limited to 1 WH... but we know we will add more.   Couldn't we have a map of webhooks which contains 1 currently and have some infra code here that doesn't need to change as we add more in?

As far as the error message is concerned... certainly "no other webhooks are supported" should be the last sentence... "right now" is implied.

Reply:
I find that premature optimization - I don't see the advantage of writing it now rather then later :) it's the same job and later we'll have more information because we'll really be at the point when adding new webhook and having multiple. And we'll need to change this code anyway otherwise the validation would be failing. Whether it should be map or slice, I would prefer to leave that decision for later, when we really need to maintain multiple. I choose the easiest option for now

I removed the right now though, thanks